src/entry.cpp

```diff
@@ -637,6 +637,7 @@ namespace {
 	std::string entry::to_string() const
 	{
 		std::string ret;
+		ret.reserve(256);
```

Comment:
It seems reasonable to only do this if there's more than one element in the entry, but this will pessimize printing an integer. What do you think about only reserving this if the type is a list or dict?